### PR TITLE
Email communication notes bug

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Changed attachment check so it wont happen for every note. Closing this PR: Fixed in https://github.com/demisto/content/pull/15961

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py [hunk 1]
```diff
@@ -142,22 +142,23 @@ def create_file_data_json(attachment):
...
         incident_id (str): The incident id.
         attachments (list): The email's attachments.
     Returns:
         The reply body and the html body.
     """
     reply_body = ''
     if notes:
+        if attachments:
+            attachment_names = [attachment.get('name') for attachment in attachments]
+            reply_body += f'Attachments: {attachment_names}\n\n'
```
Comment (reutshal) [inline Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py:155]: Lets save this as a string and add it to the reply_body where the old attachments were added so we will keep the same order.

**Review by reutshal — CHANGES_REQUESTED** [review]
Nice

File: Packs/EmailCommunication/ReleaseNotes/1_4_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### SendEmailReply
+- Fixed an issue where attachments in the layout where being attached to each one of the notes.
```
Comment (Bargenish) [inline Packs/EmailCommunication/ReleaseNotes/1_4_2.md:4]:
```suggestion
- Fixed an issue where attachments in the layout were being attached to each one of the notes. 
```

File: Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply_test.py [hunk 3]
```diff
@@ -51,6 +52,23 @@ def test_get_email_recipients(email_to, email_from, service_mail, excepted):
...
 
 
+@pytest.mark.parametrize(
+    "notes, attachments, expected_results",
+    [([{'Metadata': {'user': 'DBot'}, 'Contents': 'note1'}, {'Metadata': {'user': 'DBot'}, 'Contents': 'note2'}],
+     [{'name': 'attachment1.png'}, {'name': 'attachment2.png'}],
+     "Attachments: ['attachment1.png', 'attachment2.png']\n\nDBot: \nnote1\n\nDBot: \nnote2\n\n")
+     ]
+)
+def test_get_reply_body(mocker, notes, attachments, expected_results):
```
Comment (Bargenish) [inline Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply_test.py:62]: Please dd `Given When Then`
Also, you have only one case here so using `parametrize` is redundant, let's add a case where no attachments were added as well.

**Review by Bargenish — CHANGES_REQUESTED** [review]

**Pull request closed without merging**